onnxruntime/core/language_interop_ops/torch/torch_proxy.cc

```diff
@@ -171,6 +171,12 @@ void InvokeRunner(
   // first element.
   for (; i < static_cast<size_t>(PyTuple_Size(result_ptr.get())); ++i) {
     PyObject* dl_tensor_pointer = PyTuple_GetItem(result_ptr.get(), i);
+    if (dl_tensor_pointer == Py_None) {
+      OrtValue empty_ort_value;
+      returned_ortvalues.push_back(empty_ort_value);
+      continue;
+    }
+
     ORT_ENFORCE(Py_REFCNT(dl_tensor_pointer) == 1, "Ref count of dl_tensor_pointer should be 1.");
```

Comment:
Can the user's code cause a > 1 ref count here?

Reply:
This is a good question. The dl_tensor is created by orttraining/orttraining/python/training/ortmodule/_custom_autograd_function_runner.py (when we call to_dplack, which finally PyCapsule_New a PyObject*), so it should be only affected by our code there. User code should no increase its refcnt, but can increase the forward output torch tensor